### PR TITLE
Superuser steps

### PR DESCRIPTION
**Pull request by bucephalus**
still have pending steps for not yet fully implemented parts

### CODE REVIEW

File: features/step_definitions/patient_steps.rb [hunk 1]
```diff
@@ -17,20 +11,18 @@
 
     department = Department.find_by_name(encounter[:department]) ||
       Department.create!({:name => encounter[:department]})
-    encounter.delete(:department)
+    encounter.delete('department')
```
Comment (seshness) [inline features/step_definitions/patient_steps.rb:15]: do you need the above `end_time` line?

File: features/step_definitions/superuser_steps.rb [hunk 2]
```diff
@@ -3,78 +3,70 @@
...
+  avgs_list = avgs_list.split(/,/).map{ |avg| Integer(avg.strip)}
   pending # express the regexp above with the code you wish you had
 end
 
 Then /^I should see a "(.*?)" for average time per patient$/ do |arg1|
-  pending # express the regexp above with the code you wish you had
+  find('#average-time').text.should include arg1
 end
 
 Then /^I should not see a graph$/ do
```
Comment (seshness) [inline features/step_definitions/superuser_steps.rb:34]: I can't imagine a test with this "not see a graph" step passing on the superuser dashboard - there are `svg` elements in the haml of the page that are rendered regardless. Maybe add an `id` to the step:

``` ruby
Then /^I should not see a graph identified by "(.*?)"$/ do |graph_id|
  page.should_not have_selector("svg##{graph_id}")
end
```

File: features/step_definitions/superuser_steps.rb [hunk 3]
```diff
@@ -3,78 +3,70 @@
...
-  pending # express the regexp above with the code you wish you had
+  page.should_not have_selector("svg")
 end
 
-Given /^the average wait time is (\d+) minutes$/ do |arg1|
-  pending # express the regexp above with the code you wish you had
```
Comment (seshness) [inline features/step_definitions/superuser_steps.rb:31]: `require` statements at the top seem easier to manage..

**Comment by seshness** [comment]
:accept: conditional on comments

File: features/step_definitions/patient_steps.rb (hunk 1 above)
Comment (bucephalus) [inline features/step_definitions/patient_steps.rb:15]: I think so.  The representation we decided on for "no end time" is nil

File: features/step_definitions/superuser_steps.rb (hunk 3 above)
Comment (bucephalus) [inline features/step_definitions/superuser_steps.rb:31]: whoops  how did that get there

File: features/step_definitions/patient_steps.rb (hunk 1 above)
Comment (seshness) [inline features/step_definitions/patient_steps.rb:15]: Okay, just verified this in rails console: `Chronic::parse("")` returns `nil`. Therefore, the "no end time" case is already handled correctly, and you **don't** need the following if block:

``` ruby
if encounter['end_time'] == ""
  encounter['end_time'] = nil
end
```

**Comment by danwang** [comment]
okay I guess

**Pull request merged**